src/components/Register/index.jsx

```diff
@@ -30,6 +30,17 @@ const Register = ({ isTesting }) => {
       email: email,
       uid: uid.current,
     });
+    db.collection("soldCoupons")
+      .doc(uid.current)
+      .set({
+        compName: name,
+        "25₺": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
```

Comment:
You can also use a for loop to push the zeros in to arrays.

Reply:
`const chartStatistics = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
    db.collection("soldCoupons")
      .doc(uid.current)
      .set({
        compName: name,
        "25₺": chartStatistics,
        "50₺": chartStatistics,
        "75₺": chartStatistics,
        "100₺": chartStatistics,
        "150₺": chartStatistics,
        "200₺": chartStatistics,
      });`
what do you think of this way? Maybe we don't need to use a for loop fo this simple operation

Reply:
Well played 😂